### PR TITLE
[ARM64/Unix] LLDB

### PR DESCRIPTION
**Pull request by sdmaclea**
Patches to deal with the unfortunate fact that Arm64 LLDB is not readily available.

For ubuntu, LLDB3.9 is available in 16.10.  It is not present on earlier platforms

@jkotas

### CODE REVIEW

**Comment by mikem8361** [comment]
LGTM.

**Comment by sdmaclea** [comment]
Do I need to mention someone to have this merged?

**Comment by sdmaclea** [comment]
@janvorli Since you just merged a related patch...

File: cross/build-rootfs.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ __InitialDir=$PWD
 __BuildArch=arm
 __UbuntuArch=armhf
 __UbuntuRepo="http://ports.ubuntu.com/"
-__LLDB_Package="lldb-3.6-dev"
+__LLDB_Package=""
```
Comment (gkhanna79) [inline cross/build-rootfs.sh]: I believe this change broke our Arm32 Ubuntu build:

```
2017-02-16T00:36:40.4286270Z -- Looking for ucol_setMaxVariable - not found
2017-02-16T00:36:40.4374030Z CMake Error at src/ToolBox/SOS/lldbplugin/CMakeLists.txt:60 (message):
2017-02-16T00:36:40.4388130Z   Cannot find lldb-3.5, lldb-3.6, lldb-3.8 or lldb-3.9.  Try installing
2017-02-16T00:36:40.4406950Z   lldb-3.6-dev (or the appropriate package for your platform)
2017-02-16T00:36:40.4414420Z 
2017-02-16T00:36:40.4420130Z 
2017-02-16T00:36:40.4432910Z -- Configuring incomplete, errors occurred!
2017-02-16T00:36:40.4445300Z See also "/root/coreclr/bin/obj/Linux.arm.Checked/CMakeFiles/CMakeOutput.log".
2017-02-16T00:36:40.4456490Z See also "/root/coreclr/bin/obj/Linux.arm.Checked/CMakeFiles/CMakeError.log".
2017-02-16T00:36:40.4468580Z ~/coreclr
2017-02-16T00:36:40.4479530Z Failed to generate CoreCLR component build project!
```

If we want to nullify lddb package for arm64, it should be done conditionally and not absolutely.

Comment (gkhanna79) [inline cross/build-rootfs.sh]: @sdmaclea Can you please look into addressing this at the earliest please?

Comment (sdmaclea) [inline cross/build-rootfs.sh]: See #9625

**Comment by gkhanna79** [comment]
I think the should keep the original variable name and not add "Default" suffix since original variable name is used for pulling down packages.



Thanks,
Gaurav


-------- Original message --------

**Comment by sdmaclea** [comment]
@gkhanna79 The change that "broke" arm32 CI builds removed the **default** value for the lldb package.  Making it required for the script consumer to explicitly specify the lldb version for all platforms.  I reverted the original change and added a separate option to supress the default lldb inclusion.

**Comment by sdmaclea** [comment]
@gkhanna79 You are right.  That was a typo....

**Pull request merged**